app/services/supported_ssg_updater.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This class updates supported SSGs yaml
+class SupportedSsgUpdater
+  require 'yaml'
+
+  KEYS_TO_REMOVE = %w[upstream_version brew_url policy_version _latest_policy_version _technical_confidence].freeze
+  SUPPORTED_SSG_FILE_PATH = 'config/supported_ssg.yaml'
+  DEFAULT_SUPPORTED_SSG_FILE_PATH = 'config/supported_ssg.default.yaml'
```

Comment:
Do not redefine the same constants over and over, they're already defined in `SsgConfigDownloader` and you can reference them directly via `SsgConfigDownloader::foo`